net/frr/src/opnsense/mvc/app/controllers/OPNsense/Quagga/forms/dialogEditBFDNeighbor.xml

```diff
@@ -20,9 +25,10 @@
     <id>neighbor.multihop</id>
     <label>Multihop</label>
     <type>checkbox</type>
-    <help>multihop tells the BFD daemon that we should expect packets with TTL less than 254
-    (because it will take more than one hop) and to listen on the multihop port (4784).
-    When using multi-hop mode echo-mode will not work (see RFC 5883 section 3).
-    </help>
+    <help>Multihop tells the BFD daemon that we should expect packets with TTL less than 254 (because it will take more than one hop) and to listen on the multihop port (4784). When using multi-hop mode echo-mode will not work (see RFC 5883 section 3).</help>
```

Comment:
The  question is if this is worth the effort as it will break translations (https://github.com/opnsense/lang/blob/fca8fe48bdba970f4560a8c5874e660d4ba53ddb/en_US.pot#L47944-L47951)

Padding unfortunately is still problematic (https://github.com/opnsense/lang/issues/68), we might want to seek a solution for that, but that also means we need to change the existing translations to keep the identifiers matching.